data/USDC/data.json

```diff
@@ -17,11 +17,7 @@
       }
     },
     "base": {
-      "address": "0xd9aAEc86B65D86f6A7B5B1b0c42FFA531710b6CA",
-      "overrides": {
-        "name": "USD Base Coin",
-        "symbol": "USDbC"
-      }
+      "address": "0x833589fCD6eDb6E08f4c7C32D4f71b54bdA02913"
```

Comment:
I assume there isn't a permissionless bridge for this token since it's native, in which case you'll want to add the `"nobridge": true` tag.